cypress/support/swap-page.po.cy.ts

```diff
@@ -0,0 +1,20 @@
+export function getText(selector: string, callback: any) {
+  const text = cy.get(selector).invoke('text')
+  text.then($text => {
+    callback($text)
+  })
+}
+
+export function getTokenList(selector: string, callback: any) {
+  let arr: string[] = []
+  const listToken = cy.get(selector)
+  const len = cy.get(selector).its('length')
```

Comment:
```suggestion
  const len = listToken.its('length')
```
Why dont you reuse this variable